src/main/java/org/jenkinsci/plugins/credentialsbinding/Binding.java

```diff
@@ -77,7 +79,7 @@ public SingleEnvironment(String value, Unbinder unbinder) {
     }
 
     @Deprecated
-    public interface Environment {
+    public interface Environment extends Serializable {
```

Comment:
`UnbinderWrapper` has a [field](https://github.com/jenkinsci/credentials-binding-plugin/blob/master/src/main/java/org/jenkinsci/plugins/credentialsbinding/Binding.java#L115) of type `Environment`. `UnbinderWrapper` implements `Unbinder` and `Unbinder` implements `Serializable`. `Environment` interface does not implement `Serializable` and Findbugs complains about that with the following error:

> [INFO] Class org.jenkinsci.plugins.credentialsbinding.Binding$UnbinderWrapper defines non-transient non-serializable instance field e [org.jenkinsci.plugins.credentialsbinding.Binding$UnbinderWrapper] In Binding.java SE_BAD_FIELD